features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/EmptySessionScenario.kt

```diff
@@ -15,26 +15,24 @@ internal class EmptySessionScenario(
     eventMetadata: String
 ) : Scenario(config, context, eventMetadata) {
 
-    init {
+    override fun startBugsnag(startBugsnagOnly: Boolean) {
         config.autoTrackSessions = false
 
         val dir = File(context.cacheDir, "bugsnag-sessions")
 
-        if (eventMetadata != "non-crashy") {
-            disableAllDelivery(config)
-        } else {
+        if (startBugsnagOnly) {
             val files = dir.listFiles()
             Log.d("Bugsnag", "Empty sessions: $files")
             files.forEach { it.writeText("") }
+        } else {
+            disableAllDelivery(config)
         }
     }
```

Comment:
Existing problem, but this method doesn't call its super meaning nothing would ever be delivered

Reply:
Now added